nixd/lib/Controller/Hover.cpp

```diff
@@ -46,8 +71,7 @@ class NixpkgsHoverProvider {
     }
 
     if (Package.Description) {
-      OS << "## Description"
-         << "\n\n";
+      OS << "## Description" << "\n\n";
```

Comment:
The hunk looks to be unrelated, I don't really mind it but it will be very nice if you can remove this hunk (unit of git diff)

Reply:
I think that happened because of the autoformatter. I've reverted it.